extensions/panache/hibernate-orm-rest-data-panache/deployment/pom.xml

```diff
@@ -16,11 +16,11 @@
     <dependencies>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-hibernate-orm-rest-data-panache</artifactId>
+            <artifactId>quarkus-rest-data-panache-deployment</artifactId>
```

Comment:
@gytis can you make sure this is correct?

Reply:
These two dependencies just swapped places, so no harm there.